src/webln/NostrWeblnProvider.ts

```diff
@@ -277,10 +278,26 @@ export class NostrWebLNProvider implements WebLNProvider {
     if (options?.returnTo) {
       url.searchParams.set('return_to', options.returnTo);
     }
+
+    if (options.budgetRenewal) {
```

Comment:
a few lines above we use `options?` should `options` be required?

Reply:
Oops, I changed it back to optional. Thanks!